README.md

```diff
@@ -68,9 +68,7 @@ Run linting and tests to see if all went well:
 
     poetry run nox -r
 
-[Nox](https://nox.thea.codes/) is used for automation and standardisation of tests and linting. Any contribution **needs to pass these tests** before creating a Pull Request. The command above will check code formatting (using `flake8`), dependency safety and unit tests (using [`pytest`](https://docs.pytest.org/en/latest/) and [`coverage`](https://coverage.readthedocs.io/)). This doesn't include automatic code formatting, which is set up using [`black`](https://github.com/psf/black) to be run using:
-
-    poetry run nox -rs black
```

Comment:
I assume you removed `black` since it is run as a pre-commit - good call.

Reply:
Yes exactly -- probably should have mentioned that 👍